spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigurationClientBuilderSetup.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.config;
+
+
+import com.azure.data.appconfiguration.ConfigurationClientBuilder;
+
+public interface ConfigurationClientBuilderSetup {
```

Comment:
If we following the name convention from class should finish with substantive
```suggestion
public interface ConfigurationClientSetupBuilder {
```